timescaledb/getting-started/create-cagg/create-cagg-basics.md

```diff
@@ -1,256 +1,91 @@
+informational view:
 </highlight>
 
-When you query a continuous aggregate, the system reads and processes the much
-smaller materialized table, which has been refreshed at scheduled intervals. This
-makes querying continuous aggregates useful for speeding up dashboards, summarizing
-data sampled at high frequency, and querying downsampled data over long time periods.
-
-Additionally, querying continuous aggregates doesn't slow down `INSERT` operations,
-since data is inserted into a different hypertable than the one underlying the
-continuous aggregate.
-
-Let's examine an example of a query which would perform better on the continuous
-aggregate (`weather_metrics_daily`) than on the hypertable (`weather_metrics`).
-Here's a query which looks at how temperatures in New York have changed over the
-past 6 years. It returns the time as well as the daily maximum, minimum,
-and average temperatures for New York City between 2015 and 2021:
-
-```sql
--- Continuous aggregate query example
--- Temperature in New York 2015-2021
-SELECT bucket, max_temp, avg_temp, min_temp
-FROM weather_metrics_daily
-WHERE bucket >= '2015-01-01' AND bucket < '2021-01-01'
-AND city_name LIKE 'New York'
-ORDER BY bucket ASC;
-```
-
-Such a query executes quickly as the data for the time period in question is
-already populated in the continuous aggregate and aggregated by day. Queries like
-this are used in historical analysis, often to plot graphs about how temperature
-changes over time.
-
-### Real-time aggregation
-
-By default, continuous aggregates support real-time aggregation, which combines
-aggregated data and raw data at query time for the most up-to-date results. (You
-can turn this off if desired, but the majority of developers want this behaviour
-by default).
-
-**With real-time aggregation turned off**, continuous aggregates only return
-results for data in the time period they have materialized (refreshed). If you
-query continuous aggregates for data newer than the last materialized time, it
-does not return it or returns stale results.
-
-**With real-time aggregation turned on**, you always receive up-to-date results,
-as querying a continuous aggregate returns data that is already materialized
-combined with the newest raw data from the hypertable available at query time.
-
-Real-time aggregation gives you the best of both worlds: the performance of continuous
-aggregates and the most up-to-date data for real time queries, without the
-performance degradation of querying and aggregating all raw data from scratch.
-
-For example, consider the results of this query, which selects daily aggregates
-of all weather metrics for the past two years:
-
 ```sql
--- Real-time aggregation
-SELECT * from weather_metrics_daily
-  WHERE bucket > now() - 2 * INTERVAL '1 year'
-  ORDER BY bucket DESC;
+-- See info about continuous aggregates
+SELECT * FROM timescaledb_information.continuous_aggregates;
```

Comment:
Move code example into the highlight. I can't do it from a suggestion without breaking things unfortunately. :(

Formatting *should* work; I think we've fixed the Markdown parsing issues that were a problem earlier. If there are still problems with code block display in highlights, let me know and I'll debug.

Reply:
Sounds good! I'll move it to the highlight!